easybuild/easyconfigs/c/CGNS/CGNS-3.3.1-foss-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'CGNS'
+version = '3.3.1'
+
+homepage = 'https://cgns.github.io/'
+description = """The CGNS system is designed to facilitate the exchange of data between sites and applications,
+ and to help stabilize the archiving of aerodynamic data."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/CGNS/CGNS/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['65c55998270c3e125e28ec5c3742e15d']
```

Comment:
@iotaka Do you mind making this a SHA256 checksum?

We're trying to move towards making that a standard, and been requiring SHA256 checksums for recent pull requests.

Should be `81093693b2e21a99c5640b82b267a495625b663d7b8125d5f1e9e7aaa1f8d469`